src/index.html

```diff
@@ -7,8 +7,29 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo"><img src="images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="navList">
+          <li class="navItem"><a href="http://" class="navLink is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="navItem">
            <a href="#apple" class="navLink is-active">Apple</a>
          </li>
```
only one tag per line